client/src/components/App.jsx

```diff
@@ -1,43 +1,50 @@
 import React from 'react';
 import $ from 'jquery';
-import { FaBeer } from 'react-icons/fa';
-import { FaHeart } from 'react-icons/fa';
+
+import Photo from './Photo.jsx';
+import Content from './Content.jsx';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
-    this.handleShareButtonClick = this.handleShareButtonClick.bind(this);
+    this.state = {
+      photos: [],
+      showContent: false,
+    };
     this.handleViewPhotosButtonClick = this.handleViewPhotosButtonClick.bind(this);
   }
 
-  handleShareButtonClick() {
-
-  }
   handleViewPhotosButtonClick() {
+    this.setState({
+      photoButtonClick: !this.state.photoButtonClick
+    })
+  }
 
+  componentDidMount() {
+    $.ajax({
+      url: `/api/rooms/${this.props.roomId}/photos`,
+      method: 'GET',
+      success: (photos) => {
+        this.setState({
+          photos: photos,
+          showContent: true,
+        });
+      },
+      error: (err) => {
+        console.log(err);
+      }
+    });
   }
 
   render() {
+    const { showContent } = this.state;
+    let content = showContent ? <Content photos={this.state.photos} viewPhotosButton={this.handleViewPhotosButtonClick}/> : null;
```

Comment:
I don't understand the choice of prop name for `viewPhotosButton`. What convention are you following here?